src/schemas/PriceFeed.ts

```diff
@@ -11,6 +11,7 @@
  * Represents an aggregate price from Pyth publisher feeds.
  */
 export interface PriceFeed {
+  [x: string]: any;
```

Comment:
I am wondering about this. Did quicktype generate it like this? couldn't we have vaa like `metadata`? 🤔  in general additional fields do not break the parser here.

Reply:
good catch I'll remove it